src/apps/dashboard/modal/ready-for-loan-modal/ready-to-loan-modal-content.tsx

```diff
@@ -0,0 +1,237 @@
+import * as React from "react";
+import { FC, useCallback, useEffect, useState } from "react";
+import dayjs from "dayjs";
+import check from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-check.svg";
+import {
+  useDeleteReservations,
+  useGetReservationsV2
+} from "../../../../core/fbs/fbs";
+import { ReservationDetailsV2 } from "../../../../core/fbs/model";
+import {
+  getColors,
+  getReadyForPickup
+} from "../../../../core/utils/helpers/general";
+import {
+  useDeleteV1UserReservationsIdentifier,
+  useGetV1UserReservations
+} from "../../../../core/publizon/publizon";
+import { useText } from "../../../../core/utils/text";
+import {
+  Reservation,
+  ReservationListResult
+} from "../../../../core/publizon/model";
+import QueuedReservationsList from "./ready-to-loan-list";
+import CheckBox from "../../../../components/checkbox/Checkbox";
+import StatusCircleIcon from "../../../loan-list/materials/utils/status-circle-icon";
+import ArrowWhite from "../../../../components/atoms/icons/arrow/arrow-white";
+import { useModalButtonHandler } from "../../../../core/utils/modal";
+
+interface ReadyToLoanModalContentProps {
+  modalId: string;
+}
+const ReadyToLoanModalContent: FC<ReadyToLoanModalContentProps> = ({
```

Comment:
This Component is too large I think. Please try to break it down in smaller subcomponents / hooks